wiki/Tournaments/OMC/OMC5.md

```diff
@@ -0,0 +1,77 @@
+# osu! Fan Cafe Mapping Contest #5 
+
+![{ABBR} Logo](https://i.imgur.com/2a7gVVl.jpg)
+
+The **OMC5** (***osu! Fan Cafe Mapping Contest #5***) is a mapping contest hosted in Korea and only Koreans are permitted to participate. It is the 5th installment of the OMC.
+
+## Contest Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-08-12 ~ 2019-08-18 24:00 UTC +9 |
+| Submitting Phase | 2019-08-19 ~ 2019-09-19 24:00 UTC +9 |
+| Judging Phase | 2019-09-20 ~ 2019-09-27 24:00 UTC +9 |
+| Results Announce | 2019-09-20 |
+
+Schedules may be delayed.
+
+## Songs
+ Contestants are offered 2 songs to choose from. These maps are free from copyright issues as we have legally bought the rights to use the songs. No other songs may be mapped other than the 2 offered.
+
+|     |
+| --: |
+| 1. (to be announced) |
+| 2. Cranky - R176 |
+
+## Ruleset
+
+### Rules
+- This contest is for only osu!Standard mode.
+- Only Koreans are permitted to participate. If your profile flag is not showing you're in korea because of some reasons like foreign residence, you can participate if you're able to verify that you're a korean somehow. 
+- Contestants should only submit one difficulty. More submitting won't be considered as your entry. 
+- The difficulty shouldn't be easier than Insane, otherwise the difficulty won't be considered as an entry. 
+- Storyboard and Video may not be considered for judging.
+- Contestants may follow [Ranking Criteria](Ranking_Criteria/en.md).
+
+- You can be disqualified from contest if...
+    - ...You uploaded your entry before results announced.
+    - ...You collaborated your entry.
+    - ...You bribed Judge.
+    - ...You made an issue even if it's not related to this contest.
+    - ...Your act doesn't make sense and it's a bad influence to this contest.
+
+### Division System
+- There are 2 divisions, a 'Pro' division and a 'Rookie' division.
+- Contestants with a ranked mapset or a ranked GD of Insane or higher difficulty will be classified as a 'Pro' division contestant.
+- Contestants without a ranked mapset or a ranked GD of Insane or higher difficulty will be classified as a 'Rookie' division contestant.
+- Points 2 and 3 above will be based off of the start of the mapping date.
+- Contestants may only compete within their respective divisions.
+- Contestants may only submit one beatmap, regardless of difficulty. Following the contest, the winner will map additional difficulties to complete a full mapset. 
+
+
+## Prizes
+
+In every contest, people conquering a place on the podium are eligible to receive unique prizes created specifically for OMC winners. It is possible that these items change with every installment of the OMC.
```

Comment:
can probably delete this. I know it's in the Tournament template, but it's pretty useless/obvious information (I'll see about removing it from the template too)

Reply:
> can probably delete this. I know it's in the Tournament template, but it's pretty useless/obvious information (I'll see about removing it from the template too)

do you mean that above sentence? if so, i'll delete it yes :)

Reply:
yes